Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/ContactDiaryActivity.kt

```diff
@@ -0,0 +1,62 @@
+package de.rki.coronawarnapp.contactdiary.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.navigation.fragment.NavHostFragment
+import dagger.android.AndroidInjector
+import dagger.android.DispatchingAndroidInjector
+import dagger.android.HasAndroidInjector
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.util.di.AppInjector
+import javax.inject.Inject
+
+/**
+ * This activity holds all the contact diary fragments
+ */
+class ContactDiaryActivity : AppCompatActivity(), HasAndroidInjector {
+    companion object {
```

Comment:
Companion objects the end of the class.

Reply:
https://github.com/corona-warn-app/cwa-app-android/pull/1943
adjusted with upcoming codesmell PR